lib/dotcom/system_status/subway.ex

```diff
@@ -202,23 +217,27 @@ defmodule Dotcom.SystemStatus.Subway do
   # Service" should come after any other alerts), and then by branch
   # ID (so that, say statuses for "Green-B" should come ahead of
   # "Green-C").
+  @spec sort_branches([status_entry_group()]) :: [status_entry_group()]
   defp sort_branches(branches_with_statuses) do
     branches_with_statuses
     |> Enum.sort_by(fn %{status_entries: status_entries, branch_ids: branch_ids} ->
       {status_sort_order(status_entries), branch_ids}
     end)
   end
 
+  @spec status_sort_order([status_entry()]) :: integer()
```

Comment:
```suggestion
  @spec status_sort_order([status_entry()]) :: non_neg_integer()
```

Ha I think dialyzer didn't like this one because 0 isn't an integer!

Reply:
I may have missed it, but is dialyzer complaining? It looks to me like the build is passing. (0 is definitely an integer O_o)

Reply:
I remember you shared a dialyzer error you were wondering about before, maybe it wasn't this